packages/blocks/src/_specs/_specs.ts

```diff
@@ -196,13 +200,61 @@ const CommonFirstPartyBlockSpecs: BlockSpec[] = [
     },
     service: AttachmentService,
   },
-  EmbedFigmaBlockSpec,
-  EmbedYoutubeBlockSpec,
-  EmbedGithubBlockSpec,
+  {
+    ...EmbedFigmaBlockSpec,
+    view: {
+      ...EmbedFigmaBlockSpec.view,
```

Comment:
You can modify the `embed-figma-spec.ts` file and add the widget there instead of changing this.